### PR TITLE
Create compat package form performance exp

### PR DESCRIPTION
**Pull request by alikn**

### CODE REVIEW

File: packages-exp/performance-compat/package.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@firebase/performance-compat",
+  "version": "0.0.1",
```
Comment (Feiyang1) [inline packages-exp/performance-compat/package.json:3]:
```suggestion
  "version": "0.0.900",
```

File: packages-exp/performance-compat/package.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "build": "rollup -c",
+    "build:release": "rollup -c rollup.config.release.js",
+    "build:deps": "lerna run --scope @firebase/performance-compat --include-dependencies build",
+    "dev": "rollup -c -w",
+    "test": "run-p lint test:all",
+    "test:all": "run-p test:browser",
+    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test:all",
+    "test:browser": "karma start --single-run",
+    "test:browser:debug": "karma start --browsers Chrome --auto-watch",
+    "prepare": "yarn build:release",
```
Comment (Feiyang1) [inline packages-exp/performance-compat/package.json:24]:
```suggestion
```

File: packages-exp/performance-compat/package.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    "prettier": "prettier --write '{src,test}/**/*.{js,ts}'"
+  },
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "@firebase/app-compat": "0.x",
+    "@firebase/app-types": "0.x"
+  },
+  "dependencies": {
+    "@firebase/performance-exp": "0.0.900", 
+    "@firebase/performance-types-exp": "0.0.900",
```
Comment (Feiyang1) [inline packages-exp/performance-compat/package.json:34]:
```suggestion
```

**Review by hsubox76 — COMMENTED** [review]
To fix the changeset error, add `@firebase/performance-compat` after this line in this file:
https://github.com/firebase/firebase-js-sdk/blob/89a4f642988c1577955b868c0994235fccb34f40/.changeset/config.json#L27

File: packages-exp/performance-compat/package.json (hunk 1 above)
Comment (alikn) [reply packages-exp/performance-compat/package.json]: Done.

File: packages-exp/performance-compat/package.json (hunk 2 above)
Comment (alikn) [reply packages-exp/performance-compat/package.json]: Done.

File: packages-exp/performance-compat/package.json (hunk 3 above)
Comment (alikn) [reply packages-exp/performance-compat/package.json]: Done.

File: .changeset/orange-countries-accept.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/performance-compat": patch
```
Comment (Feiyang1) [inline .changeset/orange-countries-accept.md:2]: The changeset should be removed. exp and compat packages don't use changeset yet.

Comment (alikn) [reply .changeset/orange-countries-accept.md]: Done.

**Review by Feiyang1 — APPROVED** [review]

**Pull request merged**